pages/i18n/localization.json

```diff
@@ -24,12 +24,12 @@
             "try_it_out": "Try it out"
           },
           "headers": {
-            "header_1": "Open source",
-            "header_2": "Ease of setup",
-            "header_3": "Ready templates",
-            "header_4": "Ideal for services",
-            "header_5": "All required integrations",
-            "header_6": "Bonus: easy migration"
+            "header_open_source": "Open source",
```

Comment:
headers.header_open_source not good naming, "headers" already says that you are in it, no need to add "header"